### PR TITLE
README: PlaceholderMC -> Prism Launcher

### PR DESCRIPTION
**Pull request by NotNite**
Also cleaned it up a bit to make my Markdown linter happy.

### CODE REVIEW

**Review by zack5555 — APPROVED** [review]

File: README.md
Comment (NotNite) [inline README.md:20]: I removed the vanity URL because it was out of place and discussions about if the vanity URL should be used in other issues (it's not guaranteed to be up 100% of the time). Let me know if I should add it back.

**Review by DioEgizio — APPROVED** [review]

**Review by AshtakaOOf — APPROVED** [review]

**Review by flowln — APPROVED** [review]

**Comment by NotNite** [comment]
Resolved my comment to not block the PR since I got two approvals. Ready for merge, I think?

**Review by Scrumplex — APPROVED** [review]

**Comment by Scrumplex** [comment]
lol

**Comment by flowln** [comment]
:trollface:

**Pull request merged**
